src/loader-module.ts

```diff
@@ -2,5 +2,6 @@ import { init_as_module } from './posthog-core'
 export { PostHog } from './posthog-core'
 export * from './types'
 export * from './posthog-surveys-types'
+export { renderSurveysPreview } from './extensions/surveys'
```

Comment:
any suggestions for how to export this method for use in https://github.com/PostHog/posthog/pull/20280 without it blowing up the `loader-module` 😢

Reply:
maybe an optional PostHog init param that allows for this to load conditionally...?

Reply:
Definitely not this approach 😅 

This feels so specific to our needs, that we don't need it to be part of the main bundle.

We already can do something in the code like

```
import * from "posthog-js/dist/recorder"
```

Why don't we just have this as an exported method from the surveys loader and import it from that file in `posthog`